File: app/src/main/java/com/example/myappcontacts/presentation/contactslist/presenter/IContactsListPresenter.java
package com.example.myappcontacts.presentation.contactslist.presenter;

import java.util.UUID;

public interface IContactsListPresenter {
    void addContact();

    void deleteContact(UUID contactId);

    void loadContactsList();

    void onItemSwiped(UUID contactId);

    void onContactItemClicked(UUID contactsId);

    void onShowContactsMapClicked();
}

File: app/src/main/java/com/example/myappcontacts/presentation/contactslist/presenter/IContactsListPresenter.kt
package com.example.myappcontacts.presentation.contactslist.presenter

import java.util.UUID

interface IContactsListPresenter {
    fun addContact()

    fun deleteContact(contactId: UUID)

    fun loadContactsList()

    fun onItemSwiped(contactId: UUID)

    fun onContactItemClicked(contactsId: UUID)

    fun onShowContactsMapClicked()
}
